src/pages/PostList/components/PostListPost.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './PostListPost.scss';
+
+const PostListPost = ({ postData }) => {
+  return postData.map(feed => (
+    <div className="post" key={feed.id}>
+      <div className="writerDesktop">
+        <div className="id">
+          <img className="profileThumb" src={feed.userProfile} alt="profile" />
+          <p className="name">{feed.userName}</p>
+        </div>
+        <p className="smallFont">{feed.createdAt}</p>
```

Comment:
feed 객체를 구조분해할당하면 좀 더 간편하게 작성할 수 있겠네요!
```suggestion
  return postData.map(feed => {
    const { id, userProfile, ...생략 } = feed;
  
    return (
      <div className="post" key={id}>
        <div className="writerDesktop">
          <div className="id">
            <img className="profileThumb" src={userProfile} alt="profile" />
            ...
```